spec/features/user_management_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+feature 'User sign up' do
+
+  let(:user){build :user}
+
+  def sign_up(user)
+    visit '/users/new'
+    expect(page.status_code).to eq(200)
+    fill_in :name,     with: user.name
+    fill_in :handle,    with: user.handle
+    fill_in :email,    with: user.email
+    fill_in :password, with: user.password
+    fill_in :password_confirmation, with: user.password_confirmation
+    click_button 'Sign up'
+  end
+
+  scenario 'I can sign up as a new user' do
+    expect {sign_up(user)}.to change(User, :count).by(1)
+    expect(page).to have_content("Welcome, #{user.name}")
+    expect(User.first.email).to eq("#{user.email}")
+  end
+
+  scenario 'requires a matching confirmation password' do
+     user = create(:user, password_confirmation: 'wrong')
+     expect {sign_up(user)}.not_to change(User, :count)
+     expect(current_path).to eq('/users')
+     expect(page).to have_content 'Password does not match the confirmation'
+   end
```

Comment:
Lint/BlockAlignment: `end` at 29, 3 is not aligned with `scenario 'requires a matching confirmation password' do` at 24, 2